### ISSUE TITLE
make JSX ending bracket on a new line

### ISSUE THREAD

**Issue by fab1an** [royriojas/esformatter-jsx#67]
Moved from here: https://github.com/millermedeiros/esformatter/issues/433

Is it possible to format JSX like this:

```
<div
    className="test"
    style={{
        height: "100px"
    }}
>                               <----- on a new line
    Something
</div>
```

At the moment I get

```
<div
     className="test"
     style={{
                height: "100px"                    <----- too much indentation
            }}>                                  <--- ">" symbol not on a new line
  Something
</div>
```

I tried countless options already without result, so I ask here. I use the `esformatter-jsx` plugin.
Many thanks.

**Comment by fab1an** [comment]
Is this possible?

**Comment by royriojas** [comment]
hi @fab1an,

Sadly currently it is not possible. It might be easy to do though. PRs welcome.

**Comment by royriojas** [comment]
It was closed by #87
